instrumentation/packaging/nodejs-agent-release.txt

```diff
@@ -1 +1 @@
-v2.15.0
+v3.0.0
```

Comment:
Both versions v2.16.0 and v3.0.0 were released. Asked around and it was recommended to migrate to use v3.0.0. To do this we have to link/add the [breaking change notes](https://github.com/signalfx/splunk-otel-js/releases/tag/v3.0.0) and update our test golden file values.